vkwave/bots/core/dispatching/filters/builtin.py

```diff
@@ -119,7 +119,7 @@ async def check(self, event: BaseEvent) -> FilterResult:
 class PayloadFilter(BaseFilter):
     """Filter for message payload"""
 
-    def __init__(self, payload: Dict[str, str], json_loader: JSONDecoder = json.loads):
+    def __init__(self, payload: Dict[str, str] = None, json_loader: JSONDecoder = json.loads):
```

Comment:
wrong type. it should be `Optional[Dict[..., ...]]`

Reply:
fixed